### DISCUSSION TITLE
Should a docstring section header be allowed without a docstring summary?

### DISCUSSION THREAD

**Discussion by jorgegomezcq** [astral-sh/ruff#8394] [Q&A]
Hi,

Should a docstring section header be allowed without a docstring summary?

If ruff rule D212 is enabled, `ruff` will try to put the docstring section header on the first line thinking that this is the docstring summary.

Here is a fragment from my `git diff` of what ruff did:

```diff
-    """
-    Note:
-        unchecked Foo garbage
+    """Note:
+    unchecked Foo garbage
     """
 ```

**Comment by jorgegomezcq** [comment]
Or, stated differently. How can I avoid the above fix from occuring? I'm not sure what ruff pydocstyle rule caused it.

I'll try to trigger it again and run `ruff --show-source .` (TIL) to see if I can show myself the diff and configuration/context that would cause such a fix...

**Reply by dhruvmanila** [reply]
I think you might want to use [`D213`](https://docs.astral.sh/ruff/rules/multi-line-summary-second-line/) instead which will keep the docstring summary on the second line. Does that help your use-case?

**Reply by jorgegomezcq** [reply]
Hi, Thanks for the reply. I don't think it does because I'm using Google style docstrings and I want the docstring to be as specified in `D212` which is what the Google style docstring recommends to use... Correct me if I am wrong here.
